dashboard.md

```diff
@@ -0,0 +1,57 @@
+# Interop Dashboard
+
+Our dashboard is available at https://wpt.fyi/interop.
+
+This document explains how tests are run and how score are computed. There are several layers of async operations with varying amounts of delay, so this might also serve as a guide to identify where the problem is if the dashboard is not being updated.
+
+## Focus Area Definitions
+
+The focus areas are defined by the combination of three data sources:
+
+- [wpt-metadata](https://github.com/web-platform-tests/wpt-metadata) labels specific tests with labels like `interop-2022-subgrid`. These labels are added manually after review of the test lists. The easiest way to enumerate all labeled tests is through the [wpt.fyi API](https://wpt.fyi/api/metadata?includeTestLevel=true&product=chrome).
+- [`category-data.json`](https://github.com/web-platform-tests/results-analysis/blob/main/interop-scoring/category-data.json) defines the focus areas for each year as an identifier like `interop-2024-layout` and a set of labels like `interop-2021-flexbox`, `interop-2022-subgrid`, etc.
+- [`interop-data.json`](https://github.com/web-platform-tests/wpt.fyi/blob/main/webapp/static/interop-data.json) provides human readable names for the focus area identifiers, such as "Layout" for `interop-2024-layout`. This file also includes investigation efforts. The same data is available in [`interop-data.js`](https://wpt.fyi/components/interop-data.js).
+
+## Focus Area Scoring
+
+Scores for each focus area are computed in [results-analysis](https://github.com/web-platform-tests/results-analysis) and published as CSV ([example](https://raw.githubusercontent.com/web-platform-tests/results-analysis/gh-pages/data/interop-2024/interop-2024-experimental-v2.csv)) to GitHub pages. The CSV data and `interop-data.js` is fetched by the wpt.fyi frontend to populate the scores. 
+
+## End-to-End Pipeline
+
+The pipeline for updating scores is as follows:
+
+- All of WPT is run on Chrome, Edge, Firefox, and Safari across two different CI systems: Taskcluster and Azure Pipelines. The cadence varies:
+
+  - Chrome Canary: [every commit](https://github.com/web-platform-tests/wpt/blob/6f491c45bc5182275419be89a0820c2deaddc6b0/tools/ci/tc/tasks/test.yml#L188-L193)
```

Comment:
Do we have something to link to about how the epochs branches work or how to examine their history? At the least a link to the current state of each relevant branch would probably help.

Reply:
The entrypoint for all of that is https://github.com/web-platform-tests/wpt/blob/epochs/three_hourly/.github/workflows/epochs.yml and https://github.com/web-platform-tests/wpt/tree/epochs/three_hourly the most used of the branches. There are also tags on the pattern epochs/three_hourly/* which was the topic of https://github.com/web-platform-tests/rfcs/pull/117.